### DISCUSSION TITLE
How to run the allure source code and customize the report name from the command line. Like allure  generate -- reportName=XXXX

### DISCUSSION THREAD

**Discussion by tianyao1111** [allure-framework/allure2#2419] [Questions & Support] [answered]
### Describe the Bug

We parameterize the custom report name. But after searching for problems and forums, I couldn't find a way, only providing a location to modify the report name.

### Steps to Reproduce

We parameterize the custom report name. But after searching for problems and forums, I couldn't find a way, only providing a location to modify the report name.

### Expected Behaviour

customize the report name from the command line. Like allure  generate -- reportName=XXXX

### What Language are you using?

Java

### What Framework/Allure Integration you are using?

allure_unittest

### What version of Allure Integration you are using?

allure_unittest==0.0.2

### What version of Allure Report you are using?

2.12.0

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by baev** [comment, marked as answer]
Since [2.26.0](https://github.com/allure-framework/allure2/releases/tag/2.26.0) you can specify report name using `--name` or `--report-name` option, see https://github.com/allure-framework/allure2/pull/2229

**Reply by tianyao1111** [reply]
i got it,think you!
